src/styles/blocks/main.scss

```
@@ -0,0 +1,125 @@
+@import "../utils/variables";
+
+.main__catalog {
+  display: flex;
+  justify-content: space-between;
+  flex-wrap: wrap;
+  max-width: 944px;
+  margin: auto;
+  padding-top: 50px;
+}
+
+.card__code,
+.card_rating__reviews,
+.card__price-label {
+  font-size: 11px;
+}
+
+.card {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  flex-direction: column;
+  width: 200px;
+  padding: 29px 14px 14px;
+  background-color: #fff;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+
+  &:nth-child(4) {
+    margin-right: 0;
+  }
+
+  &:last-child {
+    margin-right: 0;
+  }
+
+  &__image {
+    width: 160px;
+    height: 134px;
+
+    &:hover {
+      transform: translateY(-14px) scale(1.21);
+      transition: 400ms;
```

```suggestion
      transition: transform 400ms, box-shadow 400ms, border-radius 400ms;
```
Include properties that you use transition for, and move transition to &__image from &:hover.